webview/src/stories/Table.stories.tsx

```diff
@@ -196,9 +203,11 @@ WithNoSortsOrFilters.args = {
 
 export const Scrolled: ComponentStory<typeof Experiments> = ({ tableData }) => {
   return (
-    <div style={{ height: 400, width: 600 }}>
-      <Experiments tableData={tableData} />
-    </div>
+    <Provider store={configureStore({ reducer: experimentsReducers })}>
+      <div style={{ height: 400, width: 600 }}>
+        <Experiments tableData={tableData} />
+      </div>
+    </Provider>
```

Comment:
Maybe `Provider` could be added with a [decorator](https://storybook.js.org/docs/react/writing-stories/decorators)?

Reply:
That is a good idea, I'll add this in my follow up PR